src/js/components/EditArticle.js

```diff
@@ -28,4 +30,4 @@ export default class EditArticle extends React.Component {
       <ArticleForm handleSubmit={ this.handleSubmit.bind(this) } ref='articleForm' />
     )
   }
-}
\ No newline at end of file
+}
```

Comment:
Yura, configure your editor to NOT automatically add new lines at the end of the document. Is it hard to just copy config from gist? It's third time I'm asking :(